README.md

```diff
@@ -218,6 +218,28 @@ WebValve.register FakeBank, url: ENV.fetch("SOME_CUSTOM_API_URL")
 WebValve.register FakeBank, url: "https://some-service.com"
 ```
 
+## Mocking dynamic URLs with wildcards
+
+If the service you are interacting with contains dynamic elements, e.g.
+an instance-specific subdomain, you can specify a wildcard in your url
+with the `*` character to match a series of zero or more
```

Comment:
is there a reason to allow for a zero-char match here? is that just a glob-compat thing in your mind? i feel like it might be more intuitive that we have to match 1 or more chars because an invalid url shouldn't be matchable.

Reply:
I think opening up the possibility of statically prefixed or only-sometimes-dynamic URLs makes sense, like `http://foo*.bar.com`, but also yeah, sort of keeping the ergonomics the same felt good. It's not our job to make people not make requests to invalid URLs, right?

Reply:
maybe `http*://foo.com` is my real winning case